📖  Remove BMO vars from docs (#328)
metal3-io/cluster-api-provider-metal3: README.md

# Cluster API Provider for Managed Bare Metal Hardware

> [![Ubuntu V1alpha5 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_ubuntu/badge/icon?subject=Ubuntu%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a4_integration_test_ubuntu/)

Reviewer: ```suggestion
[![Ubuntu V1alpha5 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_ubuntu/badge/icon?subject=Ubuntu%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_ubuntu/)
```

> [![CentOS V1alpha5 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_centos/badge/icon?subject=CentOS%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a4_integration_test_centos/)

Reviewer: ```suggestion
[![CentOS V1alpha5 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_centos/badge/icon?subject=CentOS%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_centos/)
```

> Kubernetes-native declarative infrastructure for Metal3. 

Reviewer 2: This seems a bit confusing to me

## What is the Cluster API Provider Metal3

The [Cluster API](https://github.com/kubernetes-sigs/cluster-api/)brings declarative,
Kubernetes-style APIs to cluster creation, configuration and management. The API 
itself is shared across multiple cloud providers. Cluster API Provider Metal3 is 
one of the providers for Cluster API and enables users to deploy a Cluster API based

> cluster on top of BareMetal infrastructure using Metal3. 

Reviewer 2: ```suggestion
cluster on top of bare metal infrastructure using Metal3. 
```

## Compatibility with Cluster API

| CAPM3 version | Cluster API version | CAPM3 Release |
|---------------|---------------------|---------------|
| v1alpha4      | v1alpha3            | v0.4.X        |

> | v1alpha5      | v1alpha4            | v0.5.X        |

Reviewer 2: maybe you could add another row to show that it is deprecated ?
